server/text_generation_server/models/__init__.py

```diff
@@ -193,16 +176,18 @@ def get_model(
             model_id,
             revision,
             quantize=quantize,
+            use_medusa=use_medusa,
             dtype=dtype,
             trust_remote_code=trust_remote_code,
         )
 
-    if model_type == "gpt_bigcode":
```

Comment:
How do you know if it's a true gpt2?

Reply:
I do not, I assumed the code was compatible and would work.

`model_type == "gpt2" and model_id.startwith("bigcode/")` is more correct no ?